plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaLanguageServerExtensionService.java

```diff
@@ -58,9 +62,13 @@
 /**
  * This service makes custom commands in our jdt.ls extension available to clients.
  *
- * @author Thomas Mäder
+ * @author Thomas M?der
```

Comment:
previous version was better ;)

Reply:
Thanks, returned to normal version). This is due to bug in Che, here is the issue: https://github.com/eclipse/che/issues/7528. I usually revert this, but seems forgot that time.